src/main/java/org/zalando/nakadi/service/subscription/StreamParameters.java

```diff
@@ -41,25 +43,31 @@
 
     private final Client consumingClient;
 
-    private StreamParameters(
-            final int batchLimitEvents, @Nullable final Long streamLimitEvents, final long batchTimeoutMillis,
-            @Nullable final Long streamTimeoutSeconds, @Nullable final Integer batchKeepAliveIterations,
-            final int maxUncommittedMessages, final long commitTimeoutMillis, final Client consumingClient)
-            throws UnprocessableEntityException {
-        if (batchLimitEvents > 0) {
-            this.batchLimitEvents = batchLimitEvents;
-        } else {
-            throw new UnprocessableEntityException("batch_limit can't be lower than 1");
+    private final List<EventTypePartition> partitions;
+
+    private StreamParameters(final UserStreamParameters userParameters, final long commitTimeoutMillis,
```

Comment:
Why isn't `commitTimeoutMillis` part of `UserStreamParameters`?

Reply:
Because it is not specified by the user.

Reply:
Oh yes, of course.